assignments/arrays.js

```diff
@@ -63,32 +63,71 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+//3 options to do the same thing
+for(let i = 0; i < inventory.length; i++) {
+  if (inventory[i].id === 33){
+  console.log(inventory[i])
+  }
+}
+const item = inventory[32];
+
+console.log(`Car ${item.id} is a ${item.car_year} ${item.car_make} ${item.car_model}`);
+console.log("Car 33 is a " + inventory[32].car_year + " " + inventory[32].car_make + " " + inventory[32].car_model)
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+
+
+let lastCar = inventory[inventory.length -1];
+
+console.log(`The Car make is: ${lastCar.car_make}, Car model is: ${lastCar.car_model}`);
+//console.log(inventory[inventory.length -1]);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+  for (let i = 0; i < inventory.length; i++){
+     carModels.push(inventory[i].car_model)
+  } 
+    
+console.log(carModels.sort());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+ for (let i = 0; i < inventory.length; i++){
+     carYears.push(inventory[i].car_year)
+      
+  } 
+ console.log(carYears.sort());
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
+
 let oldCars = [];
-console.log(); 
+ for (let i = 0; i < inventory.length; i++){
+     oldCars.push(inventory[i].car_year)
+   if (oldCars[i] < 2000)
+     console.log(oldCars[i]);
+      
```

Comment:
To get the number of old cars, inside the if statement we will want to push the oldCar[i] then after the loop is done we will call console.log(oldCars.length)